src/card/card-actions.jsx

```diff
@@ -24,8 +24,10 @@ let CardActions = React.createClass({
       });
     });
 
+    let mergedStyles = this.mergeAndPrefix(styles.root, this.props.style);
```

Comment:
You are right. Thanks. I just think it belongs to another PR. I'll do it if
I have any time.

On Tue, 11 Aug 2015 00:09 Olivier Tassinari <notifications@github.com>
wrote:

> In src/card/card-actions.jsx
> <https://github.com/callemall/material-ui/pull/1376#discussion_r36684926>:
>
> > @@ -24,8 +24,10 @@ let CardActions = React.createClass({
> >        });
> >      });
> >
> > +    let mergedStyles = this.mergeAndPrefix(styles.root, this.props.style);
>
> mergedStyles will not be changed here. I would say that other components
> should also use const.
> If you want to learn more about it :
> https://hacks.mozilla.org/2015/07/es6-in-depth-let-and-const.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/callemall/material-ui/pull/1376/files#r36684926>.
>